webrtc.html

```diff
@@ -3454,51 +3447,56 @@
-              "idlAttrType"><a>unsigned short</a></span>, readonly</dt>
+              "idlAttrType"><a>unsigned short</a></span>, readonly, nullable</dt>
               <dd>The port of the candidate.</dd>
               <dt><dfn><code>type</code></dfn> of type <span class=
-              "idlAttrType"><a>RTCIceCandidateType</a></span>, readonly</dt>
+              "idlAttrType"><a>RTCIceCandidateType</a></span>, readonly, nullable</dt>
               <dd>The type of the candidate.</dd>
               <dt><dfn><code>tcpType</code></dfn> of type <span class=
-              "idlAttrType"><a>RTCIceTcpCandidateType</a></span>, readonly ,
+              "idlAttrType"><a>RTCIceTcpCandidateType</a></span>, readonly,
               nullable</dt>
               <dd>If <code>protocol</code> is <code>tcp</code>,
               <code>tcpType</code> represents the type of TCP candidate.
               Otherwise, <code>tcpType</code> is <code>null</code>.</dd>
               <dt><code>relatedAddress</code> of type <span class=
-              "idlAttrType"><a>DOMString</a></span>, readonly , nullable</dt>
+              "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
               <dd>For a candidate that is derived from another, such as a relay
               or reflexive candidate, the <dfn>relatedAddress</dfn> is the IP
               address of the candidate that it is derived from. For host
               candidates, the <code>relatedAddress</code> is
               <code>null</code>.</dd>
               <dt><code>relatedPort</code> of type <span class=
-              "idlAttrType"><a>unsigned short</a></span>, readonly ,
+              "idlAttrType"><a>unsigned short</a></span>, readonly,
               nullable</dt>
               <dd>For a candidate that is derived from another, such as a relay
               or reflexive candidate, the <dfn>relatedPort</dfn> is the port of
               the candidate that it is derived from. For host candidates, the
               <code>relatedPort</code> is <code>null</code>.</dd>
+              <dt><code>ufrag</code> of type <span class=
+              "idlAttrType"><a>DOMString</a></span>, readonly</dt>
+              <dd>This carries the <code>ufrag</code> as defined in section
+              15.4 of [[!ICE]].</dd>
             </dl>
           </section>
           <section>
             <h2>Serializer</h2>
             <div>
               <p>Instances of this interface are serialized as a map with
               entries for the following attributes: candidate, sdpMid,
-              sdpMLineIndex.</p>
+              sdpMLineIndex, ufrag.</p>
             </div>
           </section>
         </div>
         <div>
           <pre class="idl">dictionary RTCIceCandidateInit {
-    required DOMString       candidate;
+             DOMString       candidate;
              DOMString?      sdpMid = null;
              unsigned short? sdpMLineIndex = null;
+    required DOMString       ufrag;
```

Comment:
This would make all existing uses of the `RTCIceCandidate` constructor or `addIceCandidate` throw exceptions. These changes could use review by a likely future implementer.